Assets/Mirror/Runtime/Transport/LLAPITransport.cs

```diff
@@ -7,10 +7,15 @@
 namespace Mirror
 {
     [Obsolete("LLAPI is obsolete and will be removed from future versions of Unity")]
-    public class LLAPITransport : TransportLayer
+    public class LLAPITransport : MonoBehaviour, ITransport
```

Comment:
Add a ToString() that returns:
"LLAPI Server {localaddress}: {localport}" if server and
"LLAPI Connected to {remoteaddress}: {remoteport}" if client

That way it shows in the NetworkManagerHUD